images/ose-alibaba-machine-controllers.yml

```
@@ -0,0 +1,30 @@
+container_yaml:
+  go:
+    modules:
+    - module: github.com/openshift/cluster-api-provider-alibaba
+content:
+  source:
+    dockerfile: Dockerfile.rhel
+    git:
+      branch:
+        target: main
+      url: git@github.com:openshift/cluster-api-provider-alibaba.git
```

the source repo is prepped now

```suggestion
        target: release-{MAJOR}.{MINOR}
      url: git@github.com:openshift-priv/cluster-api-provider-alibaba.git
```